runtime/connectors/motherduck/motherduck.go

```diff
@@ -0,0 +1,49 @@
+package motherduck
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rilldata/rill/runtime/connectors"
+	"go.uber.org/zap"
+)
+
+func init() {
+	connectors.Register("motherduck", connector{})
+}
+
+var spec = connectors.Spec{
+	DisplayName: "Motherduck",
+	Description: "Import data from Motherduck.",
+	Properties: []connectors.PropertySchema{
+		{
+			Key:         "query",
+			Type:        connectors.StringPropertyType,
+			Required:    true,
+			DisplayName: "Query",
+			Description: "Query to extract data from Motherduck.",
+			Placeholder: "select * from my_db.my_table;", // todo :: check this
```

Comment:
Could it be problematic if people try to reference the DB name in the query? Vs. just showing something like `SELECT * FROM my_table`

Reply:
I felt we should encourage people to use db name with query so added db name in placeholder query. Shouldn't be a problem if db name is used, I verified by running query with and without db name.

Reply:
Okay, that makes sense. Let's keep it